### PR TITLE
fix(website-examples): height of the images

### PR DESCRIPTION
**Pull request by morphoix**
Before: 
Screenshot from 2023-04-12 14-30-29
After: 
Screenshot from 2023-04-12 14-30-36

### CODE REVIEW

**Comment by ibgreen** [comment]
Looks good. I think it does look weird that the I3S examples all have the same thumbnail. Can we create slightly different images?

I will fix the geospatial example thumbnails.

File: website/src/components/example/styled.ts [hunk 1]
```diff
@@ -33,6 +33,10 @@ export const ExampleCard = styled.a`
     transition-property: filter;
     transition-duration: var(--ifm-transition-slow);
     transition-timing-function: var(--ifm-transition-timing-default);
+    height: 100%;
+    height: -moz-available;
+    height: -webkit-fill-available;
+    height: fill-available;
```
Comment (ibgreen) [inline website/src/components/example/styled.ts:39]: Just making sure that this does not stretch the image? It is important that we preserve the aspect ratio.

**Review by ibgreen — APPROVED** [review]

File: website/src/components/example/styled.ts (hunk 1 above)
Comment (morphoix) [reply website/src/components/example/styled.ts:39]: Should we then set the images as a background to prevent future cases?

Comment (ibgreen) [reply website/src/components/example/styled.ts:39]: Perhaps. You could also check how this is done in deck.gl examples page. You can inspect styles via the Chrome debug window.

Comment (morphoix) [reply website/src/components/example/styled.ts:39]: Deck.gl has same sized images in the gallery so it's not the case.
I can ask our designers to prepare good quality images in one size also

Comment (ibgreen) [reply website/src/components/example/styled.ts:39]: Sound good. let's cut the images to size.

**Comment by ibgreen** [comment]
Just noting:
- gltf is 512x512 
- the others are 430x430. 
- consistency would be good. I.e. what is the recommended size for the next example image?
- Nit: 512 (as a power of 2: 2^9) seems more natural than 430x430.

**Pull request merged**